mariner_proj/settings.py

```diff
@@ -252,6 +252,21 @@
 # Unique session cookie ensures that logins are treated separately for each app
 SESSION_COOKIE_NAME = "mariner"
 
+PREFERRED_COORDINATE_SYSTEMS = (
+    {
+        "name": "BNG",
+        "srid": "27700",
+        "proj4": "+proj=tmerc +lat_0=49 +lon_0=-2 +k=0.9996012717 +x_0=400000 +y_0=-100000 +ellps=airy +datum=OSGB36 +units=m +no_defs",
+        "default": True,
+    },
+    {
+        "name": "LatLong",
+        "srid": "4326",
+        "proj4": "+proj=longlat +datum=WGS84 +no_defs",
+        "default": False
```

Comment:
I believe that the default coord system for Mariner is going to be LatLong as the dataset will extend outside fo the BNG grid